go/vt/servenv/buildinfo.go

```diff
@@ -106,7 +106,10 @@ func init() {
 		goArch:             runtime.GOARCH,
 		version:            versionName,
 	}
-	sqlparser.MySQLVersion = AppVersion.MySQLVersion()
+	sqlparser.MySQLVersion, err = sqlparser.ConvertMySQLVersionToCommentVersion(AppVersion.MySQLVersion())
+	if err != nil {
+		panic(fmt.Sprintf("%s", err))
```

Comment:
Eeek... Do we have to panic? What about logging and moving on?

Reply:
Alright I'll do that

Reply:
General guideline is that vtgate should never panic.